app/javascript/budgets/modules/invoices_controller.js

```diff
@@ -67,6 +67,20 @@ window.GobiertoBudgets.InvoicesController = (function() {
 
         data = _.filter(csv, _callback(filter));
 
+        if (!data.length) {
+          // disable empty filter
+          $('#invoices-filters button[data-toggle=' + filter + ']').prop('disabled', true)
+
+          // if there's no data, get all available filters and trigger a new one
+          let filters = [];
+          $('#invoices-filters button[data-toggle]').each(function() { filters.push($(this).attr('data-toggle')) });
+          let previousFilter = (filters.indexOf(filter) > 0) ? filters[filters.indexOf(filter) - 1] : alert('Any filter has data.')
```

Comment:
You should use I18n here in the alert message.

Reply:
Oh! You're right! I forgot it, look, even I created the translations:
https://github.com/PopulateTools/gobierto/pull/1645/files#diff-c1ce275193fd9589720bb8618e801eb2R498